sdk/include/cardboard.h

```diff
@@ -42,6 +42,14 @@ typedef enum CardboardEye {
   kRight = 1,
 } CardboardEye;
 
+/// Enum to distinguish the possible orientation of the viewport
+typedef enum CardboardViewportOrientation {
+  kCardboardViewportOrientationLandscapeLeft = 0,
```

Comment:
Consider naming them with a shorter name, e.g. `kLandscapeLeft`, since the rest of the information can be inferred from the variable type.